src/Coverage.Report/CLIReportTask.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.Vstest.Coverage.Report
+{
+    using System;
+    using Microsoft.Build.Framework;
+    using Microsoft.Build.Utilities;
+
+    public class CLIReportTask : Task, ICancelableTask
+    {
+        private Trace trace;
+
+        public CLIReportTask() : this(new Trace())
+        {
+        }
+
+        public CLIReportTask(Trace trace)
+        {
+            this.trace = trace;
+        }
+
+        public override bool Execute()
+        {
+            this.trace.Log("=================================================" + Environment.NewLine +
```

Comment:
Is this a console output or a verbose information? `Log` may need verbosity enum?